en: update some steps to resolve problems (#393)
pingcap/docs-dm: en/deploy-a-dm-cluster-using-tiup.md

TiUP supports deploying DM v2.0 or later DM versions. This document introduces how to deploy DM clusters of different topologies.

> **Note:**
>

> > If the operating system for the deployment supports SELinux, make sure that SELinux is enabled.

Reviewer: ```suggestion
> If your target machine's operating system supports SELinux, make sure that SELinux is **disabled**.
```
This is a P0 translation bug. Note that your translation could impact our users/customers' production clusters and might cause severe results. Be careful!  @ireneontheway
